app/controllers/api/authentications_controller.rb

```diff
@@ -55,9 +55,9 @@ def build_additional_fields
     def validate_auth_attrs(data)
       raise 'must supply a manager resource' unless data['manager_resource']
       attrs = data.dup.except('manager_resource')
-      manager_collection, manager_id = HrefParser.parse(data['manager_resource']['href'])
-      raise 'invalid manger_resource href specified' unless manager_collection && manager_id
-      manager_resource = resource_search(manager_id, manager_collection, collection_class(manager_collection))
+      href = Href.new(data['manager_resource']['href'])
+      raise 'invalid manger_resource href specified' unless href.resource && href.resource_id
```

Comment:
typo, it's wrong in the original line 59, but let's fix this since we're updating the line anyways.

Reply:
@abellotti there's a test for this typo! Which I don't want to incorporate in this PR....I'm very happy to do a follow up